app/src/main/kotlin/com/hedvig/app/feature/crossselling/ui/CrossSellData.kt

```diff
@@ -33,6 +33,9 @@ data class CrossSellData(
 
     @Parcelize
     object Chat : Action()
+
+    @Parcelize
+    data class Web(val url: String): Action()
```

Comment:
When do we get this from backend? The `CrossSellData::from` function seems to only ever create Embark and Chat here https://github.com/HedvigInsurance/android/blob/a9570c2c033f332eb78ca14d9dd87cfdea2a6567/app/src/main/kotlin/com/hedvig/app/feature/crossselling/ui/CrossSellData.kt#L57-L59.
Is this a TODO for later, or maybe I'm missing smth